win32ss/printing/base/winspool/printers.c

```diff
@@ -834,7 +834,17 @@ IntFixUpDevModeNames( PDOCUMENTPROPERTYHEADER pdphdr )
 
     if (res)
     {
-        FIXME("IFUDMN : Get Printer Name %S\n",pi2->pPrinterName);
+        /* Check if the provided buffer is large enough */
+        WORD cbDevMode = pi2->pDevMode->dmSize + pi2->pDevMode->dmDriverExtra;
```

Comment:
Shouldn't it be better to use DWORD here instead in case the sum of these two dmSize and dmDriverExtra WORDs could overflow?